modules/solr/multi_docker_beanstalk.tf

```diff
@@ -0,0 +1,215 @@
+data "aws_elastic_beanstalk_solution_stack" "multi_docker" {
+  most_recent = true
+  name_regex  = "^64bit Amazon Linux (.*) running Multi-container Docker (.*)$"
+}
+
+resource "aws_elastic_beanstalk_environment" "solr" {
+  name                = "${var.environment_name}"
+  cname_prefix        = "${var.environment_cname}"
+  application         = "${var.application_name}"
+  version_label       = "${var.version_label}"
+  solution_stack_name = "${data.aws_elastic_beanstalk_solution_stack.multi_docker.name}"
+
+  setting {
+    namespace = "aws:autoscaling:asg"
+    name      = "MinSize"
+    value     = 3
+  }
+
+  setting {
+    namespace = "aws:autoscaling:asg"
+    name      = "MaxSize"
+    value     = 3
+  }
+
```

Comment:
Any reason you're changing these from variables to hardcoded?

Reply:
Mostly because solr is not super great at handling willy nilly autoscaling because there's a ton of internode traffic that's constantly happening. And, it feels like a 3 node cloud is maybe even overkill for a single yet to be used application. I guess I feel like this is a number that should be changed with some thought behind it, but perhaps this is a faulty way to go about that.